reference/forms/types/range.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Forms; Fields; range
+
+range Field Type
+================
+
+The ``range`` field is a slider that is rendered using the HTML5
+``<input type="range" />`` tag.
+
++-------------+---------------------------------------------------------------------+
+| Rendered as | ``input`` ``range`` field (slider in HTML5 supported browser)       |
++-------------+---------------------------------------------------------------------+
+| Inherited   | - `data`_                                                           |
+| options     | - `disabled`_                                                       |
+|             | - `empty_data`_                                                     |
+|             | - `error_bubbling`_                                                 |
+|             | - `error_mapping`_                                                  |
+|             | - `label`_                                                          |
+|             | - `label_attr`_                                                     |
+|             | - `mapped`_                                                         |
+|             | - `attr`_                                                           |
+|             | - `required`_                                                       |
++-------------+---------------------------------------------------------------------+
+| Parent type | :doc:`text </reference/forms/types/text>`                           |
++-------------+---------------------------------------------------------------------+
+| Class       | :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\RangeType` |
++-------------+---------------------------------------------------------------------+
+
+Basic Usage
+-----------
+
+    $builder->add('name', 'range', array(
+        'attr' => array(
+            'min' => 5,
+            'max' => 50
+        )
+    ));
+
+Inherited Options
+-----------------
+
+These options inherit from the :doc:`form </reference/forms/types/form>`
+type:
+
+.. include:: /reference/forms/types/options/attr.rst.inc
+
+.. include:: /reference/forms/types/options/disabled.rst.inc
+
+.. include:: /reference/forms/types/options/error_bubbling.rst.inc
+
+.. include:: /reference/forms/types/options/error_mapping.rst.inc
+
+.. include:: /reference/forms/types/options/label.rst.inc
+
+.. include:: /reference/forms/types/options/label_attr.rst.inc
+
+.. include:: /reference/forms/types/options/mapped.rst.inc
+
+.. include:: /reference/forms/types/options/read_only.rst.inc
```

Comment:
- `read_only`_ (deprecated as of 2.8) , do we need to add ? May be remove it from docs ?